csfieldguide/chapters/content/en/human-computer-interaction/sections/usability-heuristics.md

```diff
@@ -5,24 +5,24 @@
 # HCI posters for your classroom!
 
 A set of posters that support this section have been published by Jennifer Gottschalk.
-[These are available in PDF format here]().
+These are available in PDF format here:
 
 {button-link link="files/HCI-posters.pdf" file="yes" text="Download HCI Posters"}
 
 {panel end}
 
-Evaluating an interface is best done by getting feedback from having lots of potential users try it out.
+Evaluating an {glossary-link term="interface"}interface{glossary-link end} is best done by getting feedback from having lots of potential users try it out.
 However, this can be expensive and time-consuming, so HCI experts have come up with some quick rules of thumb that help us spot obvious problems quickly.
-The formal word for a rule of thumb is a *heuristic*, and in this section we will look at some common heuristics that can be used to critique an interface.
+The formal word for a rule of thumb is a *{glossary-link term="heuristic"}heuristic{glossary-link end}*, and in this section we will look at some common heuristics that can be used to critique an interface.
```

Comment:
Do you think the definition for heuristic is too long? Or do you think all of that information is needed?